src/components/content-container/styles.css

```diff
@@ -0,0 +1,23 @@
+.content-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  overflow: hidden;
+  width: 400px;
+  background-color: #fff;
+  height: 100%;
+  border-radius: 20px;
+  padding: 20px;
```

Comment:
Only a small opinion: I find the "relative" units are more flexible (`%`, `em`, `rem`, etc).   I really like `rem` units. Overall if you are thinking of multiple screen sizes.